src/backends/torch/torchlib.cc

```diff
@@ -27,11 +28,46 @@
 #endif
 
 #include "outputconnectorstrategy.h"
+#include "graph.h"
+
+#include <google/protobuf/io/coded_stream.h>
+#include <google/protobuf/io/zero_copy_stream_impl.h>
+#include <google/protobuf/text_format.h>
+
+#include "generators/net_caffe.h"
+#include "generators/net_caffe_recurrent.h"
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+
 
 using namespace torch;
 
+using google::protobuf::io::FileInputStream;
+using google::protobuf::io::FileOutputStream;
+using google::protobuf::io::ZeroCopyInputStream;
+using google::protobuf::io::CodedInputStream;
+using google::protobuf::io::ZeroCopyOutputStream;
+using google::protobuf::io::CodedOutputStream;
+
+
 namespace dd
 {
+
+  bool TorchWriteProtoToTextFile(const google::protobuf::Message& proto, std::string filename)
```

Comment:
Function name not consistent with the general style (why?)

Reply:
copy paste from file with other style, corrected